powermon/ports/usbport.py

```diff
@@ -51,6 +55,9 @@ def disconnect(self) -> None:
         self.port = None
 
     def send_and_receive(self, command: Command) -> Result:
+        if not self.is_connected():
+            log.warning("USBPort not connected")
```

Comment:
or try reconnect?

Reply:
Lets add this to the error handling work. Should we make a ticket with a list of error handling tasks?

Reply:
> Lets add this to the error handling work. Should we make a ticket with a list of error handling tasks?

makes sense, need to track stuff somewhere (or add # TODO: items